test/unittests/t_catalog_manager.cc

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
This file should be entirely removed, no?  The actual unit tests are in `t_catalog_mgr.cc`